src/sql/parts/modelComponents/card.component.html

```diff
@@ -1,33 +1,49 @@
-<div *ngIf="label" [class]="getClass()" (click)="onCardClick()" (mouseover)="onCardHoverChanged($event)" (mouseout)="onCardHoverChanged($event)">
-	<span *ngIf="hasStatus" class="card-status">
-		<div class="status-content" [style.backgroundColor]="statusColor"></div>
-	</span>
-	<span *ngIf="showRadioButton" class="selection-indicator-container">
-		<div *ngIf="showAsSelected" class="selection-indicator"></div>
-	</span>
-	<ng-container *ngIf="isVerticalButton">
-		<div class="card-vertical-button">
-			<div *ngIf="iconPath" class="iconContainer">
-				<div [class]="iconClass" [style.maxWidth]="iconWidth" [style.maxHeight]="iconHeight"></div>
+<div *ngIf="label" [class]="getClass()" (click)="onCardClick()" (mouseover)="onCardHoverChanged($event)" (mouseout)="onCardHoverChanged($event)"
```

Comment:
Did you check that these changes don't affect any current uses? I think the virtualize external data wizard is the main place it's used

Reply:
👍

Reply:
i did, works as expected